docs/microphone.rst

```diff
@@ -70,11 +117,59 @@ Functions
     * **return**: a representation of the sound pressure level in the range 0 to
       255.
 
+.. py:function:: record(duration=3000, rate=7812)
```

Comment:
Based on the implementation on the audio-recording branch, there isn't a default for duration. The examples also all pass it explicitly. If we're happy with that then this should likely be amended to:

```suggestion
.. py:function:: record(duration, rate=7812)
```